src/Features/CSharp/Portable/LanguageServices/CSharpSymbolDisplayService.SymbolDescriptionBuilder.cs

```diff
@@ -109,7 +109,7 @@ protected override void AddDescriptionForProperty(IPropertySymbol symbol)
                 else
                 {
                     var fullParts = ToMinimalDisplayParts(symbol, s_propertySignatureDisplayFormat);
-                    var neededParts = fullParts.SkipWhile(p => p.Symbol == null);
```

Comment:
I agree it's hacky... 
Your solution looks much better. Would you mind including the test from this PR to yours?